### ISSUE TITLE
Bug in modules/InboundEmail/InboundEmail.php

### ISSUE THREAD

**Issue by cartbar** [salesagility/SuiteCRM#6425]
#### Issue

In modules/Scheduler/_AddJobsHere.php, in function pollMonitoredInboxesAOP() it has the following code:
```
605                            if ($aopInboundEmailX->returnImportedEmail($msgNo, $uid, false, true, $isGroupFolderExists)) {
606                                // add to folder
607                                $sugarFolder->addBean($aopInboundEmailX->email);
```

Clearly, this is expecting that the returnImportedEmail() function will be setting $aopIboundEmailX->email, but it isn't.  Specifically, if this is a duplicate email the following code is run:

```
        } else {
            // only log if not POP3; pop3 iterates through ALL mail
            if ($this->protocol != 'pop3') {
                $GLOBALS['log']->info('InboundEmail found a duplicate email: ' . $header->message_id);
                //echo "This email has already been imported";
            }

            if (!empty($this->compoundMessageId)) {
                // return email
                $result = $this->db->query(
                    'SELECT id from emails WHERE message_id ="' . $this->compoundMessageId . '"' .
                    'AND mailbox_id = "' . $this->id . '"'
                );
                $row = $this->db->fetchRow($result);
                if (!empty($row['id'])) {
                    return $row['id'];
                }
            }

            return false;
        }
        ////	END DUPLICATE CHECK

```

The pollMonitoredInboxesAOP() function assumes that it is set and continues until it falls over

#### Possible Fix

I think that removing 

```
                if (!empty($row['id'])) {
                    return $row['id'];
                }
```

would fix the problem

#### Steps to Reproduce

1.
2.
3.
4.

#### Your Environment

* SuiteCRM Version used:
* Browser name and version (e.g. Chrome Version 51.0.2704.63 (64-bit)):
* Environment name and version (e.g. MySQL, PHP 7):
* Operating System and version (e.g Ubuntu 16.04):

**Comment by code-ph0y** [comment]
Fix to this issue will be part of PR #6707

**Comment by samus-aran** [comment]
#6707 has been merged so we can set this as Resolved Next Release
